win32ss/gdi/ntgdi/xformobj.c

```diff
@@ -243,6 +244,25 @@ XFORMOBJ_iCombineXform(
     }
 }
 
+BOOL FASTCALL
+MX_IsValid(IN PMATRIX pmx)
+{
+    FLOATOBJ foDet;
+    MulSub(&foDet, &pmx->efM11, &pmx->efM22, &pmx->efM12, &pmx->efM21);
```

Comment:
What are you checking here? If you check whether the determinant of the matrix is not equal to zero, then you could rename your function into "MX_IsInvertible".

Reply:
OK. Renamed.